python/paddle/fluid/layers/control_flow.py

```diff
@@ -976,6 +976,23 @@ def _complete(self):
                    "is_test": self.is_test})
 
 
+def assign_skip_lod_tensor_array(inputs, outputs):
+    """
+    Skip the process of copying LoDTensorArray.
+    """
+    if isinstance(inputs, (list, tuple)):
+        for i in range(len(inputs)):
+            assign_skip_lod_tensor_array(inputs[i], outputs[i])
+    elif isinstance(inputs, dict):
+        for key in _sorted(inputs):
+            assign(inputs[key], outputs[key])
+    else:
```

Comment:
Rewrite line 989 - 993:

```
elif inputs.type != core.VarDesc.VarType.LOD_TENSOR_ARRAY:
      assign(inputs, outputs)
```

Code logic is better when it is shorter :-)

Reply:
Thanks, done.